src/python/pants/backend/python/util_rules/pex_requirements.py

```diff
@@ -419,7 +419,7 @@ def _invalid_user_lockfile_error(
         # python_requirement targets that aren't in that code's resolve.
         not_in_lock = sorted(str(r) for r in user_requirements - metadata.requirements)
         yield (
-            f"- The targets use requirements that are not in the lockfile: {not_in_lock}\n"
+            f"- The targets depend on requirements that are not in the lockfile: {not_in_lock}\n"
             f"This most often happens when adding a new requirement to your project, or bumping "
             f"requirement versions. You can fix this by regenerating the lockfile with "
             f"`generate-lockfiles`.\n\n"
```

Comment:
This should probably include the resolve name: `--resolve={lockfile.resolve_name}`.

Reply:
We will have earlier said:

>         f" to install the resolve `{lockfile.resolve_name}` (from `[python].resolves`). However, "
>        "it is not compatible with the current targets because:\n\n"

Reply:
Sure: I meant that we should suggest `generate-lockfiles --resolve={lockfile.resolve_name}` here... it can save a significant amount of time.